OpenSim/Common/STOFileAdapter.h

```diff
@@ -27,9 +27,35 @@
 
 namespace OpenSim {
 
+class STODataTypeNotSupported : public Exception {
+public:
+    STODataTypeNotSupported(const std::string& file,
+                            size_t line,
+                            const std::string& func,
+                            const std::string& datatype) :
+        Exception(file, line, func) {
+        std::string msg = "Datatype '" + datatype + "' is not supported.";
+
+        addMessage(msg);
+    }
+};
+
+class STODataTypeNotFound : public Exception {
+public:
+    STODataTypeNotFound(const std::string& file,
+                        size_t line,
+                        const std::string& func) :
+        Exception(file, line, func) {
+        std::string msg = "DataType not specified in the header.";
+
+        addMessage(msg);
+    }
+};
+
 /** STOFileAdapter is a DelimFileAdapter that presets the delimiters 
 appropriately for STO files.                                                  */
```

Comment:
I think this class description (or the description for DelimFileAdapter) could explain that this file format handles a few data types, and perhaps even give a snippet of what a file might look like:

```
DataType=double
endheader
time marker1 marker2
```